react-interactive/src/Menue/Page.tsx

```diff
@@ -38,32 +37,37 @@ export interface IProps {
     /** Icon that will show next to your NavLink */
     Icon?: React.ReactNode,
     /** Name of the path or paths thats used for Dynamic Routing with Name being the root of the path*/
-    Paths?: string[]
+    Paths?: string[],
+    /** Pages where this link should remain acitive */
+    OtherActivePages?: string[]
 }
 
-
 const Page: React.FunctionComponent<IProps> = (props) => {
-
     const [hover, setHover] = React.useState<boolean>(false);
     const context = React.useContext(Context)
     const location = useLocation();
 
-    
     const isPathActive = () => {
         // Use Name as the base check and see if the current path starts with this base.
-        return location.pathname.startsWith(`${context.homePath}${props.Name}`);
+        return location.pathname.startsWith(`${context.homePath}${props.Name}`)
+            || isOtherPathActive();
+    }
+
+    const isOtherPathActive = () => {
+        if (props.OtherActivePages == undefined) return false;
+        for (const path of props.OtherActivePages!) {
+            if (location.pathname.includes(path)) return true;
+        }
     }
```

Comment:
You should return `false` if nothing matches. otherwhise it will return `true` or Nothing if there are any `OtherActivePages`